tasks/modprobe.yml

```diff
@@ -1,3 +1,22 @@
 ---
 
-# modprobe
+- name: check if modprobe is already installed | os-10
+  command: 'modprobe -V'
+  register: modprobe_installed
+  ignore_errors: True
+  changed_when: False
```

Comment:
Yeah, I don't really know why I did this. There wa a problem with installing kmod on ubuntu 16.04 but I fixed this with the apt-task in default.yml.

Anyway, I removed it.